build-logic/src/main/java/extensions/CommonExtensions.kt

```diff
@@ -38,12 +38,12 @@ internal fun CommonExtension<*, *, *, *>.setupAndroidDefaultConfig() {
 
 internal fun CommonExtension<*, *, *, *>.setupCompileOptions() {
     compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_11
-        targetCompatibility = JavaVersion.VERSION_11
+        sourceCompatibility = JavaVersion.VERSION_17
+        targetCompatibility = JavaVersion.VERSION_17
```

Comment:
Precisamos atualizar pra 17 porque a nova versão do AGP exige

Reply:
to ligado